ui-ngx/src/assets/locale/locale.constant-en_US.json

```diff
@@ -1899,6 +1899,11 @@
                 "step": "Step",
                 "min-evaluation-period": "Minimum evaluation period",
                 "max-evaluation-period": "Maximum evaluation period"
+            },
+            "default-object-id": "Default object Version (Attribute)",
+            "default-object-id-ver": {
+                "v1-0": "\"1.0\" is the object's version value if the object's version is empty.",
```

Comment:
Just "1.0" and "1.1" is fine. It is clear from the field name that this is the default object version